scripts/reliability_metrics.py

```diff
@@ -270,37 +72,71 @@ def init_states():
         StateFactory.register_state('FAULT', FaultState)
 
 
+class Utils:
+    args: argparse.Namespace = None
+    handler: logging.Handler = None
+    loggers: dict = {}
+    logger_adapters: dict = {}
+
+    @staticmethod
+    def get_arguments() -> argparse.Namespace:
+        if Utils.args is not None:
+            return Utils.args
+        parser: argparse.ArgumentParser = argparse.ArgumentParser(description='find services errors in syslog')
+        parser.add_argument('state', choices=['MASTER', 'BACKUP', 'FAULT'], help='state of the machine')
```

Comment:
this argument was not set as `required` and it does not have default value

Reply:
301a886b2aedc66845c6e3e9995990e45223861f